client/main.lua

```diff
@@ -229,81 +204,69 @@ CreateThread(function()
                             size = vec3(4, 4, 4),
                             rotation = 267.49,
                             debug = Config.DebugZones,
-                            inside = inside,
-                            onEnter = onEnter,
-                            onExit = onExit
+                            onEnter = function()
+                                lib.showTextUI(Lang:t("text.enter"))
+                            end,
+                            onExit = function()
+                                lib.hideTextUI()
+                            end,
+                            inside = function()
+                                if IsControlJustReleased(0, 38) then
+                                    lib.hideTextUI()
+                                    DoScreenFadeOut(500)
+                                    while not IsScreenFadedOut() do
+                                        Wait(10)
+                                    end
+            
+                                    SetEntityCoords(cache.ped, Config.Locations.inside.coords.x, Config.Locations.inside.coords.y, Config.Locations.inside.coords.z, false, false, false, false)
+                                    SetEntityHeading(cache.ped, Config.Locations.inside.coords.w)
+            
+                                    Wait(100)
+            
+                                    DoScreenFadeIn(1000)
+                                end
+                            end
                         })
                     end
 
-                elseif #(pos - vector3(Config.Locations["inside"].coords.x, Config.Locations["inside"].coords.y, Config.Locations["inside"].coords.z)) < 1.5 then
-                    local function onEnter()
-                        lib.showTextUI(Lang:t("text.go_outside"))
-                    end
-                     
-                    local function onExit()
-                        lib.hideTextUI()
-                    end
-
-                    local function inside()
-                        if IsControlJustReleased(0, 38) then
-                            lib.hideTextUI()
-                            DoScreenFadeOut(500)
-                            while not IsScreenFadedOut() do
-                                Wait(10)
-                            end
-    
-                            SetEntityCoords(cache.ped, Config.Locations["outside"].coords.x, Config.Locations["outside"].coords.y, Config.Locations["outside"].coords.z, false, false, false, false)
-                            SetEntityHeading(cache.ped, Config.Locations["outside"].coords.w)
-    
-                            Wait(100)
-    
-                            DoScreenFadeIn(1000)
-                        end
-                    end
-                    
+                elseif #(pos - vector3(Config.Locations.inside.coords.x, Config.Locations.inside.coords.y, Config.Locations.inside.coords.z)) < 1.5 then
```

Comment:
You can use `Config.Locations.inside.coords.xyz` instead, same for all the other instances

Reply:
Would you like it done that way or just .coords?